packages/core/src/contracts/tcr/parameterizer.ts

```diff
@@ -525,7 +492,7 @@ export class Parameterizer extends BaseWrapper<ParameterizerContract> {
    * @param propID ID of prop to check
    */
   public async getPropProcessBy(propID: string): Promise<Date> {
-    const expiryTimestamp = await this.instance.getPropProcessBy.callAsync(propID);
+    const [, , , , , expiryTimestamp] = await this.instance.proposals.callAsync(propID);
```

Comment:
gross

Reply:
if you have a better way of doing this, let me know. This is generally how we've been doing it though.